### PR TITLE
SONIC QOS YANG - Remove qos tables field value reference format

### PR DESCRIPTION
**Pull request by AshokDaparthi**
Depends on https://github.com/Azure/sonic-utilities/pull/1626
Depends on https://github.com/Azure/sonic-swss/pull/1754

QOS tables in config db used ABNF format i.e "[TABLE_NAME|name] to refer fieldvalue to other qos tables.

Example:
Config DB:
"Ethernet92|3": {
"scheduler": "[SCHEDULER|scheduler.1]",
"wred_profile": "[WRED_PROFILE|AZURE_LOSSLESS]"
},
"Ethernet0|0": {
"profile": "[BUFFER_PROFILE|ingress_lossy_profile]"
},
"Ethernet0": {
"dscp_to_tc_map": "[DSCP_TO_TC_MAP|AZURE]",
"pfc_enable": "3,4",
"pfc_to_queue_map": "[MAP_PFC_PRIORITY_TO_QUEUE|AZURE]",
"tc_to_pg_map": "[TC_TO_PRIORITY_GROUP_MAP|AZURE]",
"tc_to_queue_map": "[TC_TO_QUEUE_MAP|AZURE]"
},

This format is not consistent with other DB schema followed in sonic.
And also this reference in DB is not required, This is taken care by YANG "leafref".

Removed this format from all platform files to consistent with other sonic db schema.
Example:
"Ethernet92|3": {
"scheduler": "scheduler.1",
"wred_profile": "AZURE_LOSSLESS"
},

Dependent pull requests: 
https://github.com/Azure/sonic-buildimage/pull/7752  - To modify platfrom files 
https://github.com/Azure/sonic-buildimage/pull/7281 - Yang model 
https://github.com/Azure/sonic-utilities/pull/1626   - DB migration 
https://github.com/Azure/sonic-swss/pull/1754    - swss change to remove ABNF format

### CODE REVIEW

**Comment by anshuv-mfst** [comment]
Hi @neethajohn , @stephenxs- could you please review this PR. This is related to https://github.com/Azure/sonic-utilities/pull/1626 and https://github.com/Azure/sonic-swss/pull/1754, thanks.

**Comment by stephenxs** [comment]
> Hi @neethajohn , @stephenxs- could you please review this PR. This is related to [Azure/sonic-utilities#1626](https://github.com/Azure/sonic-utilities/pull/1626) and [Azure/sonic-swss#1754](https://github.com/Azure/sonic-swss/pull/1754), thanks.

Sure. Will review it.
Btw, is there a PR for the regression test? I think we need to update regression test as well

**Comment by neethajohn** [comment]
> > Hi @neethajohn , @stephenxs- could you please review this PR. This is related to [Azure/sonic-utilities#1626](https://github.com/Azure/sonic-utilities/pull/1626) and [Azure/sonic-swss#1754](https://github.com/Azure/sonic-swss/pull/1754), thanks.
> 
> Sure. Will review it.
> Btw, is there a PR for the regression test? I think we need to update regression test as well

Are you referring to the sonic-mgmt test? I don't think there is a PR yet. @AshokDaparthi, please update that as well

**Comment by AshokDaparthi** [comment]
> > > Hi @neethajohn , @stephenxs- could you please review this PR. This is related to [Azure/sonic-utilities#1626](https://github.com/Azure/sonic-utilities/pull/1626) and [Azure/sonic-swss#1754](https://github.com/Azure/sonic-swss/pull/1754), thanks.
> > 
> > 
> > Sure. Will review it.
> > Btw, is there a PR for the regression test? I think we need to update regression test as well
> 
> Are you referring to the sonic-mgmt test? I don't think there is a PR yet. @AshokDaparthi, please update that as well

@neethajohn - I will explore what to update, i am not aware of sonic-mgmt tests.

**Comment by AshokDaparthi** [comment]
@stephenxs @neethajohn - Regarding sonic-mgmt test cases, i am not doing any new functionality, Do you want me to change existing test cases to remove the ABNF format reference?

**Comment by neethajohn** [comment]
> @stephenxs @neethajohn - Regarding sonic-mgmt test cases, i am not doing any new functionality, Do you want me to change existing test cases to remove the ABNF format reference?

Yes

**Review by stephenxs — DISMISSED** [review]

**Comment by AshokDaparthi** [comment]
@stephenxs @neethajohn - Updated the sonic-mgmt test cases https://github.com/Azure/sonic-mgmt/pull/3824

**Comment by stephenxs** [comment]
@AshokDaparthi can you fix the conflicts

**Comment by neethajohn** [comment]
@AshokDaparthi, please resolve the merge conflicts

**Comment by neethajohn** [comment]
@AshokDaparthi, how are these changes verified? Also you have only mentioned #7281 as a dependency in this PR. That contains only the SCHEDULER, WRED_PROFILE and QUEUE related changes. What about the other MMU tables (BUFFER PROFILE, BUFFER_POOL etc)? since this PR contains changes for all MMU/QOS attributes

**Comment by AshokDaparthi** [comment]
@neethajohn - https://github.com/Azure/sonic-buildimage/pull/7281 this PR is YNAG PR. For ALL QOS YANG PR's are blocked by this DB format. These YNAG PR's for buffers will are opened by broadcom else once this PR's are merged i will push sonic YNAG PR's. 
Regarding testing, i get sonic-utilities , sonic-swss. sonic-buildiimage changes and created image and tested manually broadcom platform.

**Comment by neethajohn** [comment]
@AshokDaparthi , can you please mention the merge order that you had discussed with Prince in this PR

**Comment by AshokDaparthi** [comment]
@neethajohn - Merged order is below 
1#https://github.com/Azure/sonic-swss/pull/1754
2# https://github.com/Azure/sonic-mgmt/pull/3824
3# submodule update of swss 
4# Merge this pull request                                                                                                                                                               
Can you help with merge first 2.

**Review by stephenxs — APPROVED** [review]

**Review by neethajohn — APPROVED** [review]

**Pull request merged**
